project/test/suites/test_breadcrumb.gd

```diff
@@ -49,13 +49,15 @@ func test_breadcrumb_default_values() -> void:
 
 
 func test_breadcrumb_timestamp_is_set_automatically() -> void:
-	var time_before: float = Time.get_unix_time_from_system()
+	const time_tolerance: float = 0.05  # 50 ms
```

Comment:
Wouldn't something like 1ms be enough to account for floating-point precision issues?